### PR TITLE
SUL23-457: Update entity reference card lists to be in a <ul><li>

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Changed the entity reference list structure to a `ul li` from a `div`

# Review By (Date)
- 5/22

# Urgency
- normal

# Steps to Test

1. Open the testing site.
2. The library pages have a `Meet the Staff` list that shows the change: https://su-library-git-sul23-457-card-lists-stanford-libraries.vercel.app/libraries/branner-earth-sciences-library-map-collections
3. Another example is on this page: https://su-library-git-sul23-457-card-lists-stanford-libraries.vercel.app/research-support/subject-specialists

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SUL23-457](https://stanfordits.atlassian.net/browse/SUL23-457)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by imonroe — APPROVED** [review]
Works as described on the preview site.  LGTM.

**Comment by pookmish** [comment]
This assumes they are using them as such. Unrelated content might not be appropriate as list items. Something to inform them

**Comment by jenbreese** [comment]
> This assumes they are using them as such. Unrelated content might not be appropriate as list items. Something to inform them

Thanks @pookmish, I'm writing them a note.  So they could do one off cards and they would be a one item list which might be weird?

**Comment by pookmish** [comment]
> Thanks @pookmish, I'm writing them a note. So they could do one off cards and that would be a one item list which might be weird?

Yeah, but you can do some simple logic to change the markup if there's only 1 item. I'm just thinking if there's a situation where they aren't appropriate to be in a list with 2+ other items. Could it happen, yeah.. Not sure what the context might be for that though.

File: src/components/paragraph/stanford-entity.tsx [hunk 1]
```diff
@@ -60,15 +60,24 @@ const StanfordEntity = async ({
...
+            )
+            }
+          </ul>
+          )
+          :
+          (
+            <div className="mx-auto w-full">
+              <NodeCardDisplay node={entities[0]} h3Heading={headline}/>
+            </div>
+          )
```
Comment (jenbreese) [inline src/components/paragraph/stanford-entity.tsx:78]: @pookmish I changed it to this so the single entity is a `div` and the more than one is a list.  What do you think of this route?

**Pull request closed without merging**